src/Umbraco.Core/Models/Rdbms/UmbracoDeployDependencyDto.cs

```diff
@@ -0,0 +1,30 @@
+using Umbraco.Core.Persistence;
+using Umbraco.Core.Persistence.DatabaseAnnotations;
+
+namespace Umbraco.Core.Models.Rdbms
+{
+    [TableName("umbracoDeployDependency")]
+    [PrimaryKey("id")]
+    [ExplicitColumns]
+    internal class UmbracoDeployDependencyDto
+    {
+        [Column("id")]
+        [PrimaryKeyColumn(Name = "PK_umbracoDeployDependency")]
+        public int Id { get; set; }
```

Comment:
yeah, we can do that, just wasn't sure if those were supposed to be the unique keys in this situation.